scripts/proxy-wix-ui-icons/proxy-wix-ui-icons.js

```diff
@@ -27,6 +29,8 @@ const proxyWixUiIconsCommon = (moduleName, iconsDir) => {
   const systemIconsFilesContentMap = prepareSystemIconsContentMap(moduleName);
   /* content of index.js file with named exports of all icons */
   const indexFileContent = prepareIndexFileContentForNamedExport(generalIconsFilesContentMap);
+  /* content of index.d.ts file with named exports of all icons */
+  const indexDefinitionFileCOntent = prepareIndexDefinitionFileContentForNamedExport(generalIconsFilesContentMap);
```

Comment:
```suggestion
  const indexDefinitionFileContent = prepareIndexDefinitionFileContentForNamedExport(generalIconsFilesContentMap);
```

on a side note, are functions names of this length really necessary? why not `createIndexFile` or something else shorter?

Reply:
I basically mimicked existing code and added the word definition to wherever I needed to emit a `.d.ts`, I don't truly think it's necessary I just don't like steering away from a project's conventions, even with good intentions.

Reply:
I can change it to whatever you like if you want :)